pages/affinity-diagramming.md

```diff
@@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
 
 ## Reasons to use it
 
-Quick and collaborative approach for drawing insights from qualitative data.
+To draw out insights from qualitative data quickly and collaboratively.
```

Comment:
Oh—no, I meant that in this instance, a PR that wasn't looking like it was touching card content , you as a member of the team could have tagged me because you know who I am. For general/public contributions, I think the "Copy edit" tag (added by our team while reviewing, or the contributor themselves) is sufficient, no need to add any more documentation.